R/evaluator.R

```diff
@@ -16,27 +16,28 @@ Evaluator <- R6::R6Class(
     eval_params = NULL,
     rmd_options = list(digits = 2, sigfig = FALSE,
                        options = list(scrollX = TRUE, scrollCollapse = TRUE)),
-    show = TRUE,
+    rmd_show = TRUE,
```

Comment:
I think I'll update this to `.rmd_show` rather than `.show` when adding `.` to our args. Does that sound good to you?

Reply:
Sounds good